python/Ganga/GPIDev/Adapters/IGangaFile.py

```diff
@@ -71,6 +72,15 @@ def put(self):
         """
         raise NotImplementedError
 
+    def copyTo(self, targetPath):
+        """
+        Copy a the file to the local storage using the get mechanism
+        Args:
+            targetPath (str): Target path where the file is to copied to
+        """
+        self.localDir = targetPath
```

Comment:
[I've added a note in the main conversation about this as I feel we need to be sure of the API here and this will need more discussion]